src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -112,11 +169,15 @@ void PocoHTTPClient::MakeRequestInternal(
                 poco_request.set(header_name, header_value);
 
             Poco::Net::HTTPResponse poco_response;
+
+            Stopwatch watch(CLOCK_MONOTONIC);
```

Comment:
Let's use default timer, that is already CLOCK_MONOTONIC.

Reply:
Fixed